src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PremisesSummaryTest.kt

```diff
@@ -67,6 +67,112 @@ class PremisesSummaryTest : IntegrationTestBase() {
     }
   }
 
+  @Test
+  fun `Get all CAS3 Premises returns a bedspace count of 3 when there are 3 bedspaces with no end date and one 1 with an end date in the past`() {
```

Comment:
I will suggest to change the method name to:
Get all CAS3 Premises returns bedspace count as expected when there is an archived bedspace